APCAppCore/APCAppCore/Library/Scheduler/APCScheduler.m

```diff
@@ -1123,29 +368,33 @@ - (void) fetchTasksAndSchedulesFromServerAndThenUseThisQueue: (NSOperationQueue
         else
         {
             /*
-             Bounce over to the Sage SDK's thread, call the server, and then come
+             Bounce over to the Bridge SDK's thread, call the server, and then come
              back to our thread a while later.
              */
-            [SBBComponent (SBBScheduleManager) getSchedulesWithCompletion: ^(SBBResourceList *schedulesList,
-                                                                             NSError *errorFetchingSchedules)
+            NSDate *futureDate = [[NSDate date] dateByAddingDays:3]; // TODO: store this variable somewhere, can it be in the api?
```

Comment:
cool. I figure this can be tweaked whenever AppCore is updated to the appropriate BridgeSDK that works the new way.